using-timescaledb/distributed-hypertables.md

```diff
@@ -103,6 +103,90 @@ the data node's data is outside the retention window,
 [`drop_chunks`][drop_chunks] can be used to delete the data on the data
 node.
 
+### Replication Factor for Distributed Hypertables [](replication-factor)
+
+A hypertable can have a *replication factor* that tells how many
+replicas of chunks that should be created. The default replication
+factor is 1, which means that each row of data is written to a single
+chunk. If you set the replication factor of a hypertable to a higher
+value, chunks will be replicated and each row of data will be written
+to that many chunks.
+
+With a replication factor of one, data is written to the data nodes
+using a one-phase commit, which basically means that every transaction
```

Comment:
This is incorrect, we never use one-phase commit, unless this is forced in a GUC setting.

Also, replication factor has nothing to do with transactional semantics. Even if replication_factor is 1, a transaction can involve writes to multiple chunks on different data nodes and still need to governed by two-phase commit.

One-phase commit would only be possible if the transaction involves only one data node, which, again, has nothing to do with replication factor.

Reply:
Right, I was thinking a single data node, but that is not what it is.